spec/index.adoc

```diff
@@ -2921,8 +2943,8 @@ else
   S with
     messages = Older_messages · Younger_messages
     balances[M.receiver] =
-      S.balances[M.receiver] +
-      ((if Is_response then MAX_CYCLES_PER_RESPONSE else MAX_CYCLES_PER_MESSAGE) - Cycles_used)
+      (S.balances[M.receiver] + (if Is_response then MAX_CYCLES_PER_RESPONSE else MAX_CYCLES_PER_MESSAGE))
+      - min (R.cycles_used, (if Is_response then MAX_CYCLES_PER_RESPONSE else MAX_CYCLES_PER_MESSAGE))
```

Comment:
@bogwar please double check this part

Reply:
Yeap, that's correct (modulo the fact that we don't distinguish between requests and responses when it comes down to limiting how much we reserve/allow for execution)